shanoir-ng-datasets/src/main/java/org/shanoir/ng/dataset/model/Dataset.java

```diff
@@ -81,7 +81,8 @@
 		@JsonSubTypes.Type(value = SpectDataset.class, name = "Spect"),
 		@JsonSubTypes.Type(value = StatisticalDataset.class, name = "Statistical"),
 		@JsonSubTypes.Type(value = TemplateDataset.class, name = "Template"),
-		@JsonSubTypes.Type(value = BidsDataset.class, name = "BIDS") })
+		@JsonSubTypes.Type(value = BidsDataset.class, name = "BIDS"),
+		@JsonSubTypes.Type(value = BidsDataset.class, name = "Measurement") })
```

Comment:
MeasurementDataset instead of BidsDataset I guess ?

Reply:
totally right!